src/stage1/migrate_info/balena_cfg_json.rs

```diff
@@ -230,3 +195,34 @@ impl BalenaCfgJson {
         &self.file
     }
 }
+
+fn check_api(api_endpoint: &str) -> Result<()> {
+    let ping_endpoint = format!("{}/ping", api_endpoint);
+    let res = Client::builder()
+        .build()
+        .upstream_with_context("Failed to create https client")?
+        .get(&ping_endpoint)
+        .send()
+        .upstream_with_context(&format!(
+            "Failed to send https request url: {}",
+            &api_endpoint
```

Comment:
I think using `ping_endpoint` here would be a little, super tiny bit more informative:

```suggestion
            "Failed to send https request url: {}",
            &ping_endpoint
```